src/electronic-ids/pcsc/LatEIDIDEMIAv2.cpp

```diff
@@ -32,10 +32,20 @@
 using namespace pcsc_cpp;
 using namespace electronic_id;
 
+// Workaround gcc 14.2 bug
+struct VectorComparator
+{
```

Comment:
Shouldn't the constructor be `constexpr` as well?

```suggestion
{
    constexpr VectorComparator() noexcept = default;

```